dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
+
+properties:
+  label:
+    type: string
+    description: |
+      Human readable string describing the sensor. It can be used by an
+      application to identify this sensor and distinguish it from other
```

Comment:
What exactly do we mean by "used by an application".  Are we saying, use it to determine the `struct device *`?

Reply:
No, not to determine the `struct device *`.

https://github.com/zephyrproject-rtos/zephyr/pull/49294#discussion_r950504646

Reply:
why can't application use DT for this purpose?

```
zephyr,user {
	lid-accelerometer = &accx;
	base-accelerometer = &accy;
};
```

(or use appropriate bindings)

Reply:
@MaureenHelm @galak ping